### PR TITLE
add Nokia Edge Network Controller helm charts

### PR DESCRIPTION
**Pull request by mercelistom**
Adding Nokia Edge Network Controller to the applications

sign-off-by: Tom Mercelis <tom.mercelis at nokia.com>

### CODE REVIEW

**Comment by architag21** [comment]
Reviewed and approved..

**Review by architag21 — APPROVED** [review]
Looks okay to me.

File: applications/nokia-edge-network-controller/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+SPDX-License-Identifier: Apache-2.0
```
Comment (architag21) [inline applications/nokia-edge-network-controller/README.md:1]: '#' is missing here.
# SPDX-License-Identifier: Apache-2.0
# Copyright (c) 2022 Nokia Corporation

**Review by architag21 — CHANGES_REQUESTED** [review]

File: applications/nokia-edge-network-controller/README.md (hunk 1 above)
Comment (mercelistom) [reply applications/nokia-edge-network-controller/README.md:2]: I've put the license and copyright info in text block, # would make it a very large header

**Review by abjyoti — APPROVED** [review]
LGTM

**Pull request merged**